website/docs/enterprise/installation/airgap.mdx

```diff
@@ -423,24 +415,27 @@ charts: pull-charts push-charts
 
 images:
 	kubectl get pods --all-namespaces -o jsonpath="{.items[*].spec['containers','initContainers'][*].image}" \
-    |tr -s '[[:space:]]' '\n' | sort | uniq | grep -vE 'kindest|kube-(.*)|etcd|coredns' | xargs -L 1 -I {} ./image-pull-push.sh {} $(REGISTRY)
+    |tr -s '[[:space:]]' '\n' | sort | uniq | grep -vE 'kindest|kube-(.*)|etcd|coredns' | xargs -L 1 -I {} ./image-sync.sh {} $(REGISTRY)
 	kubectl get microvmmachinetemplates --all-namespaces -o jsonpath="{.items[*].spec.template.spec.kernel.image}"|tr -s '[[:space:]]' '\n' \
-    | sort | uniq | xargs -L 1 -I {} ./image-pull-push.sh {} $(REGISTRY)
+    | sort | uniq | xargs -L 1 -I {} ./image-sync.sh {} $(REGISTRY)
 
 pull-charts:
 	curl -L https://s3.us-east-1.amazonaws.com/weaveworks-wkp/releases/charts-v3/$(WGE_CHART) --output  $(WGE_CHART)
 
 push-charts:
 	helm cm-push -f $(WGE_CHART) $(PRIVATE_HELM_REPO_NAME)
 ```
-where `image-pull-push.sh` is similar to
+where `image-sync.sh` is similar to
 
 ```shell
-docker pull $1
-docker tag $1 $2/$1
-docker push $2/$1
+#--dest-tls-verify=false to work over http with the registry in this guide, if you have an https registry, you could remove this setting
+skopeo copy docker://$1 docker://$2/$1 --dest-tls-verify=false --preserve-digests --multi-arch=all
 ```
```

Comment:
Let's remove the --dest-tls-verify=false, if they need to not verify, they can add that...but it should never be the default.

Reply:
done, yes it is a bit weird to have it